content/measure.html

```diff
@@ -59,7 +59,7 @@ <h1 class="hero-title">Measure</h1>
               <h2>Audit your web page</h2>
             </div>
             <p>
-              Provide any URL, and Lighthouse will run a series of audits generating a report of how well the page did.
+              Provide any URL, and web.dev will run a series of audits using <a href="https://developers.google.com/web/tools/lighthouse/" class="gc-analytics-event" data-category="web.dev" data-label="measure, Lighthouse link" data-action="click">Lighthouse</a>. We'll generate a report of how well you did.
```

Comment:
Should this be a `class="external" target="_blank"` like the other audit guide links that link to `developers.google.com`?

Reply:
We tried that but there were too many external link icons.